tests/lib/linter.js

```diff
@@ -18,14 +17,13 @@
  * @private
  */
 function compatRequire(name, windowName) {
-    if (typeof window === "object") {
-        return window[windowName || name];
+    if (typeof window === "object") { // eslint-disable-line no-undef
+        return window[windowName || name]; // eslint-disable-line no-undef
```

Comment:
is the inline comment really required? IMO, it should not report.(it can be an enhancement to the rule).

Reply:
Yes, it's required. Also see: https://github.com/eslint/eslint/issues/8549